devices/HumanDynamicsEstimator/HumanDynamicsEstimator.cpp

```diff
@@ -1231,40 +1235,41 @@ bool HumanDynamicsEstimator::attach(yarp::dev::PolyDriver* poly)
             return false;
         }
 
-        // Check the interface
-        if (pImpl->iHumanState->getNumberOfJoints() == 0
+        //TODO Check the interface
+        int count = 1000000;
```

Comment:
Just a proposal, instead of using this counter, couldn't we simply add here a timeout that should ensure having the interface ready?

Reply:
With https://github.com/robotology/human-dynamics-estimation/pull/309/commits/cd9f6cd60f285b0ddfa342f9db5a02b1a19ac5dc I switched to using a timeout in seconds (fort the time being set to 10s and hard-coded).